wiki/start/a-simple-game.md

```diff
@@ -5,605 +5,1091 @@ redirect_from:
   - /dev/simple_game/
 ---
 
-Before diving into the APIs provided by libGDX, let's create **a very simple "game"**, that touches each module provided by the framework, to get a feeling for things. We'll introduce a few different concepts without going into unnecessary detail.
+Let's make a game! Game design is hard, but if you break up the process into small, achievable goals, you'll be able to produce wonders. In this simple game tutorial, you will learn how to make a basic game from scratch. These are the essential skills that you will build on in future projects.
 
-{% include setup_flowchart.html current='3' %}
+{% include embed-gwt.html dir='a-simple-game' %}
 
-In the following, we'll look at:
+As you can see with the live demo, we're going to make a basic game where you control a bucket to collect water droplets falling from the sky. There is no score or end goal. Just enjoy the experience! Here are the steps that we will use to split up the game design process:
 
-  * Basic file access
-  * Clearing the screen
-  * Drawing images
-  * Using a camera
-  * Basic input processing
-  * Playing sound effects
+  * [Prerequisites](/wiki/start/a-simple-game.md#prerequisites)
+  * [Loading Assets](/wiki/start/a-simple-game.md#loading-assets)
+  * [The Game Life Cycle](/wiki/start/a-simple-game.md#the-game-life-cycle)
+  * [Rendering](/wiki/start/a-simple-game.md#rendering)
+  * [Input Controls](/wiki/start/a-simple-game.md#input-controls)
+  * [Game Logic](/wiki/start/a-simple-game.md#game-logic)
+  * [Sound and Music](/wiki/start/a-simple-game.md#sound-and-music)
+  * [Further Learning](/wiki/start/a-simple-game.md#further-learning)
+  * [Full Example Code](/wiki/start/a-simple-game.md#full-example-code)
 
-## Project Setup
-Follow the steps in the [Generating a Project](/wiki/start/project-generation) guide. In the following, we will use these settings:
+## Prerequisites
+There are a few things that you need to do before you begin this tutorial.
 
-  * Application name: `drop`
-  * Package name: `com.badlogic.drop`
-  * Game class: `Drop`
+  * Make sure to follow the [setup instructions](/wiki/start/setup) on libgdx.com to configure Java and your IDE. You need to know how to create a project and run it with the appropriate Gradle commands.
+  * Create a project in GDX-Liftoff with the following settings:
+    * Project Name: Drop
+    * Package: com.badlogic.drop
+    * Main Class: Main
+  * Include the Core and Desktop platforms. You may include others, however they will not be discussed in this tutorial.
```

Comment:
It might be worth it to mention that including some other platforms might mess with the build in various ways. "If you add more platforms or choose extra options, and then hit unexpected problems early on, try making the project again and just including Core and Desktop."

Reply:
I included something along these lines. Thanks.